core/token_chain_validation.go

```diff
@@ -92,120 +92,115 @@ func (c *Core) ValidateTokenChain(userDID string, tokenInfo *wallet.Token, token
 	//This for loop ensures that we fetch all the blocks in the token chain
 	//starting from genesis block to latest block
 	for {
-		//GetAllTokenBlocks returns next 100 blocks and nextBlockID of the 100th block,
+		//GetAllTokenBlocks returns all blocks and nextBlockID of the last block,
```

Comment:
did you update the getalltokenblocks logic to ensure all blocks are returned and isnt limited at first 100 blocks in case if TC is longer than 100?